### ISSUE TITLE
Feature Request: status bar (like title, but at the bottom)

### ISSUE THREAD

**Issue by buanzo** [IngoMeyer441/simple-term-menu#16]
Hi @IngoMeyer441 ! First of all, thanks for all the things you have been adding.

I have now implemented in mailman3commander a couple of the new features, particularly the one I asked for originally (accept_keys).

I have implemented some accept_keys that are actually global commands, that are not related to the specific menu item currently selected. And I would like to easily show that. Of course, I could use the title property, but I visualize the global shortcuts being indicated at the bottom (I am an old-school console user, and I am designing this software for console people).

At first I thought I could adapt the preview feature, but as I use it on another part of the application, it might confuse users.

Do you think a status_bar, and maybe a status_above_preview (or status_below_preview) boolean could be implemented?

Thanks!

**Comment by IngoMeyer441** [comment]
Hey @buanzo,
sounds like a useful feature. :+1: simple-term-menu already has a search bar (and an optional search hint) at the bottom but I think we could place the status bar below it. Probably the status bar should have a background color (like in tmux and other terminal applications)?
I don't know yet when I have time to work on it, but I definitely think I should add this feature. It could be useful for a couple of people and probably also in my own projects.

**Comment by IngoMeyer441** [comment]
The current `develop` has an implementation of a `status_bar` and `status_bar_style` parameter. Could you please try this out?
```bash
pip install git+https://github.com/IngoMeyer441/simple-term-menu@develop
```
`status_bar` can be a string, a multiline string, a list of strings or a callable which takes the currently selected meny entry and returns a status bar string.

**Comment by buanzo** [comment]
Acknowledged. I am testing now, will provide feedback in a moment.

**Comment by buanzo** [comment]
Works wonderfully!

![image](https://user-images.githubusercontent.com/553167/109037247-4985fb00-76a9-11eb-859d-d868a552760e.png)

I checked status_bar_style for positioning options, but I see none. In any case, this is great. I will implement a callable later. 

Thank you so much!

**Comment by IngoMeyer441** [comment]
I have added a `status_bar_below_preview` option so the status is either directly above or below the preview window.

**Comment by IngoMeyer441** [comment]
The new feature is included in the [v0.11.0](https://github.com/IngoMeyer441/simple-term-menu/releases/tag/v0.11.0) release.

I plan to publish a v1.0.0 release in the next minutes which has some few breaking changes, so you may want to stick with the v0.11.0 release (these changes are documented in the [README file](https://github.com/IngoMeyer441/simple-term-menu/blob/develop/README.md)).

Feel free to reopen if the status bar still lacks some important feature or you find bugs.
